Tools/cases_generator/analyzer.py

```diff
@@ -1280,6 +1288,70 @@ def analyze_forest(forest: list[parser.AstNode]) -> Analysis:
         instructions, uops, families, pseudos, labels, opmap, first_arg, min_instrumented
     )
 
+#Simple heuristic for size to avoid too much stencil duplication
+def is_large(uop: Uop) -> bool:
+    return len(list(uop.body.tokens())) > 80
```

Comment:
This is too low, to be useful. For example, `_BINARY_OP_ADD_FLOAT` is 89 tokens. It should be register allocated, but instead it spills all the time because it's limited to `r2_1`. I've inspected the code for nbody and it spills unecessarily. We should up the limit.

I suggest increasing the limit to 110.